tests/checkpoint/test_checkpoint.py

```diff
@@ -239,12 +239,16 @@ def test_checkpoint_filesystem_round_trip_adds_ids(
         asset = ds.add_csv_asset(asset_name, "my_file.csv")  # type: ignore[arg-type]
 
         bc1 = asset.add_batch_definition(batch_definition_name_1)
-        suite1 = ExpectationSuite(suite_name_1)
-        vc1 = ValidationDefinition(name=validation_definition_name_1, data=bc1, suite=suite1)
+        suite1 = context.suites.add(ExpectationSuite(suite_name_1))
```

Comment:
All the test changes are just using managers/factories to persist objects since they will now fail in these workflows if not attached to stores/context